src/calendar/example/index.ts

```diff
@@ -6,6 +6,24 @@ import Calendar from '../../calendar/Calendar';
 import Checkbox from '../../checkbox/Checkbox';
 import dojoTheme from '../../themes/dojo/theme';
 
+interface MyWidgetProperties {
```

Comment:
Was this intentional, or just for testing?

Reply:
Definitely a mistake :joy:. Thanks for catching that!